tracing-opentelemetry/src/layer.rs

```diff
@@ -102,7 +102,6 @@ pub(crate) fn build_span_context(
                 let mut result = sampler.should_sample(
                     builder.parent_context.as_ref(),
                     trace_id,
-                    span_id.clone(),
```

Comment:
Why's this included in a release PR?

Reply:
Hm yeah this was an impl change for the opentelemetry 0.6.0 upgrade, I can pull out to separate PR

Reply:
👍 Pulled these changes into https://github.com/tokio-rs/tracing/pull/745